src/useCases/createUser.ts

```diff
@@ -79,6 +79,16 @@ export default async (
   currentUser: Partial<User>,
   userDetails: Partial<User>
 ): PromiseResult<void> => {
+  const tmpUserDetails: Partial<User> = {}
```

Comment:
I wonder if this logic should happen before validating fields on the page. Let's say that you insert white-space characters into a text box and then try saving the form. The value of the field is trimmed here and it uses an empty string in the query string.
What do you think?

Reply:
I think we have discussed this offline a little bit before. If we were to throw a validation error, the user will be more annoyed than "Ah, sure I will remove the white space". I doubt there is a situation when adding a whitespace at the either end of the string would have been intended? They would most likely have made a mistake. Unless they are trying to input a password of course.